railties/test/generators/active_storage_install_generator_test.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "generators/generators_test_helper"
+require "generators/active_storage/install/install_generator"
+
+class ActiveStorage::Generators::InstallGeneratorTest < Rails::Generators::TestCase
+  include GeneratorsTestHelper
+
+  setup do
+    Rails.application = Rails.application.class
+    Rails.application.config.root = Pathname(destination_root)
+  end
+
+  teardown do
+     Rails.application = Rails.application.instance
+   end
+
+  test "creates migrations" do
+    run_generator_instance
+    assert_migration "db/migrate/create_active_storage_tables.active_storage.rb"
+  end
+
+  test "creates fixtures" do
+    run_generator_instance
+    assert_file "test/fixtures/active_storage/blobs.yml"
+  end
+
+  private
+    def run_generator_instance
```

Comment:
I see this same method in the ActionText generator test, maybe could be refactored?
https://github.com/rails/rails/blob/3eb5f34ec33d0d20b23050f451ed18611996aced/railties/test/generators/action_text_install_generator_test.rb#L89-L97

Reply:
👍 Make sense, I'll change. Thank you

Reply:
I ended up removing it, there is already a `with_database_configuration` method in a included helper that suits the needs of the tests (The added stubs is not required as I don't need to assert on what commands where ran)